package/MDAnalysis/analysis/atomicdistances.py

```diff
@@ -145,6 +146,10 @@ class AtomicDistances(AnalysisBase):
 
 
     .. versionadded:: 2.5.0
+
+    .. versionchanged:: 2.9.0
+        Implementation of `Results` into the class
+        for application in parallelization.
```

Comment:
Need to be clearer about what was done:
```suggestion
        Changed storage of results from :attr:`results` to 
        :attr:`results.distances`. This fix **breaks old 
        code** that expects to find the data array in 
        :attr:`results`. 
```
... assuming a breaking change.